### ISSUE TITLE
Unicode characters don't render

### ISSUE THREAD

**Issue by ekolis** [kvirc/KVIrc#1787]
I get weird text like this:

> 14:33:48] <+parabolize> What you are experiencing is El NiÃ±o. The trade currents in the Pacific ocean have changed direction. Extreme El NiÃ±os may be linked to global warming. https://en.wikipedia.org/wiki/El_Ni%C3%B1o_Southern_Oscillation#On_global_warming

I know this sort of thing can happen when I'm using a font that lacks the appropriate characters, but I'm using Arial as my global application font... pretty sure Arial is a Unicode font!

Running revision 30289 - kind of old, but I keep forgetting I'm running the beta and I can actually update! ;) Maybe I should file a ticket for an auto-updater...

**Comment by DarthGandalf** [comment]
What is 30289?

Did you select UTF-8 as encoding?

**Comment by un1versal** [comment]
@ekolis feel free to update yourself, see https://github.com/kvirc/KVIrc/wiki/Downloading-KVIrc's-nightly-source-or-binaries.
If you have UTF-8 enabled it should work what you describe indicates you dont have UTF-8 setup.

Also read https://github.com/kvirc/KVIrc/wiki/Submitting-a-bug-report#information-required-when-making-a-bug-report-via-github-issues and provide the details requested.

thx.

**Comment by ekolis** [comment]
@DarthGandalf I think 30298 was the "revision number" from the Help > About > Executable Information screen - figured that would be the most unique identifier for the app! I upgraded to the latest version (though there might be a new one since then) and it looks like the revision number format changed, because now it says "git-5903-g0850e2b".

@un1versal How do I get UTF-8 set up? I thought it was included on all modern operating systems? Or is this something I need to configure somewhere else?

**Comment by un1versal** [comment]
The link at  https://github.com/kvirc/KVIrc/wiki/Submitting-a-bug-report#information-required-when-making-a-bug-report-via-github-issues provides simple instructions on what information to provide. though git-5903-g0850e2b is pretty recent.

For utf-8 setup, look at **server encoding/textencoding fileds** on the server config dialog.
![capture](https://cloud.githubusercontent.com/assets/3521959/12012249/80ff70c8-ace4-11e5-9e9e-0d11cf4b04ad.PNG)

That willl usually do the trick but if it doesnt try changing KVIrc settings also.in.
![capture1](https://cloud.githubusercontent.com/assets/3521959/12012257/b508cf72-ace4-11e5-8687-1de14f6ff9da.PNG)

**Comment by DarthGandalf** [comment]
> I think 30298 was the "revision number" from the Help > About > Executable Information screen

Even during SVN times the number was never bigger than 6445

**Comment by ekolis** [comment]
@un1versal: ah, I think that helps! The servers were set to "use system encoding", which I assumed meant "use the encoding that Windows uses" (and which I assumed in turn to be some sort of Unicode encoding, because I'm not running Windows 95 ;) ), rather than "use the encoding specified in the settings dialog" as the tooltip explained. Maybe that setting ought to be reworded?

Anyway, after noting that, I went on to check the "system" encoding, and found it set to "use language encoding"... I'm pretty sure that means "use the encoding of my language, in this case English", so I changed it to UTF-8... hopefully that will help! Need to get someone to send me some accented characters :) Why would it default to limiting your character set, anyway? That doesn't seem very useful...
